conda/testing/helpers.py

```diff
@@ -305,7 +305,7 @@ def _patch_for_local_exports(name, subdir_data, channel, index):
     subdir_data._mtime = float("inf")
 
 
-@lru_cache(maxsize=None)
+# @lru_cache(maxsize=None)
```

Comment:
We need a fresh `index_r_1` to fix a test run order dependency on test_mkl. This is only called a few times.